README.md

```diff
@@ -27,9 +27,13 @@ Check the below video for a quick demo of XC3.
 
 - Provides cost optimization recommendation workflow without exposing your private information
```

Comment:
Before this, let's talk about there are two different modes of deployment in XC3

Reply:
Thanks for the feedback. You're right there two modes of deployment in XC3, So ?

Reply:
@faheem-khau9 I've added details about 'dev' and 'prod' environment, please have a look. Thanks